PocketKit/Sources/PocketKit/Home/RecommendationOverflowButton.swift

```diff
@@ -8,7 +8,7 @@ class RecommendationOverflowButton: UIButton {
 
         configuration = .plain()
         configuration?.contentInsets = .zero
-        configuration?.image = UIImage(asset: .verticalOverflow)?
+        configuration?.image = UIImage(asset: .alert)
```

Comment:
Once we want to re-use this button w/ different icons we can make the image a param to the initializer